examples/with-postgres/pages/_document.tsx

```diff
@@ -0,0 +1,18 @@
+import { Html, Head, Main, NextScript } from 'next/document'
```

Comment:
Seems to be unnecessary, let's remove this file.

Reply:
Removing...